.ci/run-opensearch.sh

```diff
@@ -54,14 +55,23 @@ END
 END
 ))
 
+OPENSEARCH_REQUIRED_VERSION="2.12.0"
+# Starting in 2.12.0, security demo configuration script requires an initial admin password
+COMPARE_VERSION=`echo $OPENSEARCH_REQUIRED_VERSION $OPENSEARCH_VERSION | tr ' ' '\n' | sort -V | uniq | head -n 1`
+if [ "$COMPARE_VERSION" != "$OPENSEARCH_REQUIRED_VERSION" ]; then
+  CREDENTIAL="admin:admin"
+else
+  CREDENTIAL="admin:myStrongPassword123!"
+fi
+
```

Comment:
This won't work for 2.13, will it?

Reply:
It should since admin credentials changes were released in 2.12. If you are referencing script execution in general, it works as expected returning `admin:admin` for versions <2.12 and `admin:myStronPassword123!` for those >= 2.12

Reply:
Ok, but wouldn't it be simpler to just set `OPENSEARCH_USERNAME` and `OPENSEARCH_PASSWORD` in `env:` in the workflow?

Reply:
yes, it should be, however on line 64 we make a status check call that would require the correct credentials to be passed.